tfdeterminism/patch.py

```diff
@@ -70,14 +70,20 @@ def _patch():
   if re.match("(1\.(14|15)|2\.0)", tf_version):
     os.environ['TF_CUDNN_DETERMINISTIC'] = '1'
     _patch_bias_add()
-    # Apply the fused softmax/cross-entropy patch here
+    _patch_fused_softmax_cross_entropy()
+    print("TensorFlow version %s has been patched "
+          "using tfdeterminism version %s" %
+          (tf_version, __version__), file=sys.stderr)
+  elif re.match("2\.1|2\.2", tf_version):
```

Comment:
I want to deprecate patch, not update it, and no longer advertise it in the documentation. If folks want determinism in TensorFlow version 2.1 or 2.2 then they should be using `enable_determinism`. Not adding this does not break anything because there was no patch available for TensorFlow versions 2.1 and 2.2 before.